portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,16 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Displays a random Napoleon Dynamite quote on the page.
```

Comment:
This function isn't "displaying" the quote - the html file is. A more accurate name would be `getRandomGreeting`